.github/SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of `fakerphp/faker` have active support:
+
+- `^1.23.0`
+
+## Unsupported Versions
+
+The following versions of `fakerphp/faker` have reached their end of life:
+
+- `<1.23.0`
+
+## Reporting a Vulnerability
+
+If you believe that you have found a security vulnerability, please send an email to ?. Ensure to include all details required to understand the severity of the issue.
```

Comment:
Who would like to receive emails about security issues related to `fakerphp/faker`?

Reply:
I can check if we can register a fqdn for this? Setting up a mailbox feels better to link it directly to the lib. What do you think?

Reply:
you can send em to me if you want. you could also just list multiple emails to send to - i've done that before with projects. hello at gjcampbell.co.uk

Reply:
I prefer an email address that is not tied to a single person.

Reply:
probably we should list multiple email addresses, then

Reply:
That's easily doable with Cloudflare email routing, that's probably fine?

Reply:
I've send a message in slack to discuss a domain name registration, I can register it and add you all to the cloudflare dashboard for it.

Reply:
@bram-pkg 

Maybe a single email address will suffice, perhaps `security@fakerphp.org`?